installation/install.sh

```
@@ -1781,6 +1782,28 @@ install_system_libs(){
+        return
+    fi
+    trap 'traplogger $LINENO "$BASH_COMMAND" $?'  DEBUG
+    if [ ! -z "$ZSTACK_YUM_REPOS" ]; then
+        yum --disablerepo="*" --enablerepo="$ZSTACK_YUM_REPOS" clean metadata >/dev/null 2>&1
+        echo yum install --disablerepo="*" --enablerepo="$ZSTACK_YUM_REPOS" -y key-manager >>$ZSTACK_INSTALL_LOG
+        yum install --disablerepo="*" --enablerepo="$ZSTACK_YUM_REPOS" -y key-manager >>$ZSTACK_INSTALL_LOG 2>&1
+        [ $? -ne 0 ] && fail "failed to install key-manager from repo(s) $ZSTACK_YUM_REPOS"
+    fi
+
+    if systemctl list-unit-files crypto-daemon.service 2>/dev/null | grep -q '^crypto-daemon.service'; then
+        systemctl enable crypto-daemon.service >>$ZSTACK_INSTALL_LOG 2>&1 && systemctl start crypto-daemon.service >>$ZSTACK_INSTALL_LOG 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)